### ISSUE TITLE
Multi Index Search

### ISSUE THREAD

**Issue by Wulfheart** [valeriansaliou/sonic#244]
Is it possible to search through multiple indexes in one query?

**Comment by valeriansaliou** [comment]
No, unfortunately its not possible. 1 query = 1 (collection, bucket) tuple

**Comment by Wulfheart** [comment]
Thanks. 
Is it at least possible to get the relevance of a result?

**Comment by valeriansaliou** [comment]
Most relevant results come first, there is no returned relevance index though.

**Comment by Wulfheart** [comment]
As currently searching multiple indexes is not yet supported I'd like to do some separate queries and merge them afterwards. Is it possible to get the relevance of a result so that I can order all results from the different queries? Or do you know any other way I can achieve this behavior?

**Comment by valeriansaliou** [comment]
No, it's not possible to get the relevance from search results, so that use case would not be possible.

**Comment by mkuchak** [comment]
This feature would be fantastic. Is there a plan for that?

**Comment by valeriansaliou** [comment]
Not ATM, but I'm keeping this open as a feature request :) Thanks a ton!

**Comment by wpatsche** [comment]
> This feature would be fantastic. Is there a plan for that?

I was searching for this feature, it would be great to have this because with just one call to Sonic I can solve a big problem and I believe that the performance will be greater than multiple calls

**Comment by valeriansaliou** [comment]
Thank you all for commenting! Looks like this is in huge demand :) Will make it a priority whenever I resume my work on Sonic.

**Comment by Borber** [comment]
> Thank you all for commenting! Looks like this is in huge demand :) Will make it a priority whenever I resume my work on Sonic.

One more person to wait for

**Comment by RemiBardon** [comment]
Hello! Just wanted to let you know that Sonic now computes a proper score for each result. We haven’t decided the syntax yet but Sonic will likely expose a new command to get scores as part of responses (at some point in the future).
